casual/casual-caller/src/main/java/se/laz/casual/connection/caller/TransactionLess.java

```diff
@@ -62,17 +63,17 @@ public Optional<DiscoveryReturn> discover(ConnectionFactoryEntry connectionFacto
     {
        try(CasualConnection connection = connectionFactoryEntry.getConnectionFactory().getConnection())
        {
-          LOG.finest(() -> "domain discovery for all known services/queues will be issued for " + connectionFactoryEntry );
-          LOG.finest(() -> "all known services/queues being, services: " + cachedItems.get(CacheType.SERVICE) + " queues: " + cachedItems.get(CacheType.QUEUE));
+          LOG.log(DEBUG,() -> "domain discovery for all known services/queues will be issued for " + connectionFactoryEntry );
+          LOG.log(DEBUG,() -> "all known services/queues being, services: " + cachedItems.get(CacheType.SERVICE) + " queues: " + cachedItems.get(CacheType.QUEUE));
           return Optional.of(connection.discover(UUID.randomUUID(),
                   cachedItems.get(CacheType.SERVICE),
                   cachedItems.get(CacheType.QUEUE)));
        }
        catch (ResourceException e)
        {
           connectionFactoryEntry.invalidate();
-          LOG.warning(() -> "failed domain discovery for: " + connectionFactoryEntry + " -> " + e);
-          LOG.warning(() -> "services: " + cachedItems.get(CacheType.SERVICE) + " queues: " + cachedItems.get(CacheType.QUEUE));
+          LOG.log(WARNING,() -> "failed domain discovery for: " + connectionFactoryEntry + " -> " + e,e);
+          LOG.log(WARNING,() -> "services: " + cachedItems.get(CacheType.SERVICE) + " queues: " + cachedItems.get(CacheType.QUEUE),e);
```

Comment:
`e`  should not be on both, as it will result in two log entries of the same stacktrace. Please remove from this second log call.